EvalView/templates/EvalView/pairwise-assessment-document-newui.html

```diff
@@ -237,9 +248,9 @@
     candidateTextA.removeClass('selected');
     candidateTextB.removeClass('selected');
 
-    if (candidateNum === 1 && score1Input.val() === '60') {
+    if ((candidateNum === 1 && score1Input.val() >= 60) ||
+        (candidateNum === 2 && score2Input.val() >= 60)) {
         score1Input.val('50');
```

Comment:
I wonder if we also could be changing the scores depending on the `sourceVisible` in this function. If I understand correctly, we will not have that distinction only in scores for the final preference decision, but we could keep the information per paragraph depending whether the source was visible at the time of clicking or not.

Reply:
Thank you for your feedback. In the “submit_form” function, I have already adjusted the scores based on the state of sourceVisible. When the source text is visible, I adjust the scores to 41, 51, or 61. Otherwise, I set the scores to 40, 50, or 60.
![image](https://github.com/AppraiseDev/Appraise/assets/42863309/548239db-df4c-43f4-8a03-6255fdeeafb4)